joomla/joomla-cms
# Fix tooltip for subform table layout, #39848

Pull Request for Issue #39848 .

### Summary of Changes
Fixing the tooltip for table layout with `groupByFieldset="true"` parameter


### Testing Instructions

Create a subform, somewhere:
```xml
<field type="subform" name="teest" label="Subform"
  groupByFieldset="true" multiple="true"
  layout="joomla.form.field.subform.repeatable-table">
  <form>
    <fieldset name="set1" label="set1" description="Set 1 description">
        <field type="text" name="text" label="text"/>
        <field type="textarea" name="bigtext" label="bigtext"/>
    </fieldset>
    <fieldset name="set2" label="set2" description="Set 2 description">
        <field type="text" name="text2" label="text2"/>
        <field type="textarea" name="bigtext2" label="bigtext2"/>
    </fieldset>
  </form>
</field>
```

And check the table header, there should be tooltip with a description text on hover.


### Actual result BEFORE applying this Pull Request
Tooltip icon is visible but no text


### Expected result AFTER applying this Pull Request
Tooltip icon is visible, and there a text


### Link to documentations
Please select:
